br.app.pw3270.terminal.yml

```diff
@@ -0,0 +1,43 @@
+app-id: br.app.pw3270.terminal
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: pw3270
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: lib3270
+    buildsystem: autotools
```

Comment:
for autotools the declaration is not necessary

Reply:
I can remove it but I *think* the declaration works as some kind of documentation.

Reply:
it's fine to leave it.